lib/src/main/java/com/cloudbees/groovy/cps/impl/CastBlock.java

```diff
@@ -4,20 +4,22 @@
 import com.cloudbees.groovy.cps.Continuation;
 import com.cloudbees.groovy.cps.Env;
 import com.cloudbees.groovy.cps.Next;
-import java.util.Collections;
+import com.cloudbees.groovy.cps.sandbox.CallSiteTag;
+import java.util.Collection;
 import java.util.List;
 import org.codehaus.groovy.runtime.InvokerInvocationException;
 import org.codehaus.groovy.runtime.ScriptBytecodeAdapter;
 
-public class CastBlock implements Block {
+public class CastBlock extends CallSiteBlockSupport {
```

Comment:
I verified that this does not cause problems deserializing `CastBlock`s serialized before this change.